README.md

```diff
@@ -17,25 +17,28 @@ https://discord.gg/5JmwPJf
 - OpenGL (Version >= 3.3)
 
 #### Building
+Building of this project is a simple endeavour, requiring only a few commands.
 
-- Either clone https://github.com/GentenStudios/quartz-engine or fork and then clone your fork. You will have to either clone with the
-``--recursive`` option, or you can run: ``git submodule update --init`` after cloning.
+Initially, a [clone](https://help.github.com/en/articles/cloning-a-repository) of the repository will be required, however, if you plan on making edits first, make sure you [fork](https://help.github.com/en/articles/fork-a-repo) the repo beforehand, and clone that.
```

Comment:
If a user is not contributing then they can download and extract the zip file easier than learning the command line, if they are contributing they need to fork no matter what. To keep things simple let's direct developers to fork and clone (and they should know how) and testers to download the zip.

Reply:
The best way to request changes on GitHub is to fork a project, make changes, and then make a pull request. It’s better to remove the part about cloning.

What I mean is, make installation instructions separate from contributing instructions. It’s very confusing otherwise.